.github/workflows/smoke.yaml

```diff
@@ -11,11 +11,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up Go
```

Comment:
I assume this is because the build is happening inside docker?

Reply:
Yep!